teleporters/init.lua

```diff
@@ -38,7 +44,7 @@ function teleporters.find_safe(_pos)
 	return _pos
 end
 
-dofile(minetest.get_modpath("teleporters").."/legacy.lua")
+dofile(minetest.get_modpath(minetest.get_current_modname()).."/legacy.lua")
```

Comment:
The mod name is (probably) not going to change, and mods may depend on the mod with name `teleporters`.
```suggestion
dofile(minetest.get_modpath("teleporters").."/legacy.lua")
```

~~Maybe I should rename the repository…~~

Reply:
The "Problem" is when downloaded from git, it is renamed to "teleporters-master".

So I prefer to get the name with the function `minetest.get_current_modname()`

Reply:
I've just seen that this repository is actually a modpack. You don't even need to change the name because the mod is always called teleporters, so I think there is no problem.

Afaik, the node registration does not work with a different Mod name, too. Furthermore, hard-coding the Mod name requires (slightly) less code than using the `get_current_modname` function call.